merge_request_approvals.go

```diff
@@ -35,31 +35,36 @@ type MergeRequestApprovalsService struct {
 //
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/merge_request_approvals.html#merge-request-level-mr-approvals
+// https://docs.gitlab.com/ee/api/merge_request_approvals.html#change-configuration
```

Comment:
Please remove this line, don't think we need more then one link for one struct.

Reply:
No problem, I'll do it when I create the new option struct.